.github/workflows/tests.yml

```
@@ -78,8 +87,13 @@ jobs:
       - name: Run go vet
         run: go vet -v ./...
 
-      - name: Run tests
+      - name: Run Artifactory / Build tools tests
+        run: go test -v github.com/jfrog/jfrog-cli --timeout 0 --test.${{ matrix.suite }}=true --jfrog.url='http://localhost:8081/' --jfrog.user='admin' --jfrog.password='password' --ci.runId=${{ runner.os }}-${{ matrix.suite }}
```

Remove default values
```suggestion
        run: go test -v github.com/jfrog/jfrog-cli --timeout 0 --test.${{ matrix.suite }}=true --ci.runId=${{ runner.os }}-${{ matrix.suite }}
```